packages/components/molecules/f-searchbox/src/components/Form.vue

```diff
@@ -4,6 +4,7 @@
         :action="formUrl"
         :class="$style['c-search']"
         method="post"
+        novalidate
```

Comment:
Possible potential improvement – could you add this with JS, so that if for some reason the JS errored, then it could use base HTML validation (if combined with `required` attribute for instance on the form field?

Reply:
Sure, will take a look. 👍